app/apollo/models/resource.schema.js

```diff
@@ -69,6 +69,7 @@ ResourceSchema.statics.getIds = async(ids)=>{
 ResourceSchema.statics.getByClusterIds = async(clusterIds)=>{
   return await this.find({ cluster_id: { $in: clusterIds } });
 };
+ResourceSchema.index({ cluster_id: 'text', selfLink: 'text', 'searchableData.searchableExpression': 'text' });
```

Comment:
Hi Harish, could you please move `searchableExpression` as root field of resource, and dump `cluster_id`,the whole json of `searchableData` into searchableExpression, That will allow as search text for Kind, name, api version, subscription_id, and any annotation in searhableData.

Reply:
nm, might be fine in searchableData.